contracts/eden/src/inductions.cpp

```diff
@@ -29,10 +52,20 @@ namespace eden
       check_valid_induction(induction);
       validate_profile(new_member_profile);
 
-      induction_tb.modify(induction_tb.iterator_to(induction), eosio::same_payer, [&](auto& row) {
-         row.new_member_profile = new_member_profile;
-         row.endorsements = {};
-      });
+      induction_tb.modify(induction_tb.iterator_to(induction), eosio::same_payer,
+                          [&](auto& row) { row.new_member_profile = new_member_profile; });
+
+      reset_endorsements(induction.id);
+   }
+
+   void inductions::reset_endorsements(uint64_t induction_id)
```

Comment:
I'm assuming that `endorsements` is supposed to be the number of endorsements remaining and that it will be decremented with each endorsement.  If that's true, then it needs to be reset as well.

Reply:
(no, my idea was to have `endorsements` to be a fixed number of how many endorsements this induction process needs)